packages/cli/src/commands/site/index.ts

```diff
@@ -68,25 +68,15 @@ const command: CommandModule<CliCommonArgs, CommonSiteArgs> = {
       .command(version)
       .command(deploy)
       .check((argv) => {
-        if (application != null && argv.application !== application) {
-          consola.debug(
-            `Overriding "application" from config file with ${argv.application}`,
-          );
-        }
-
-        if (foundryUrl != null && argv.foundryUrl !== foundryUrl) {
-          consola.debug(
-            `Overriding "foundryUrl" from config file with ${argv.foundryUrl}`,
-          );
-        }
-
+        argv.foundryUrl = argv.foundryUrl.replace(/\/$/, "");
```

Comment:
This also seems like a weird side effect to put in a check method and not obvious when this would run in the yargs lifecycle. Can we do this in a more explicit yargs method for example coerce or something else?